site/docs/install-ubuntu.md

```diff
@@ -3,25 +3,29 @@ layout: documentation
 title: Installing Bazel on Ubuntu
 ---
 
-# <a name="ubuntu"></a>Install Bazel on Ubuntu
+# <a name="ubuntu"></a>Install Bazel on Linux
 
-Supported Ubuntu Linux platforms:
+Supported Linux platforms:
```

Comment:
We support Windows 7+, macOS 10.11, Ubuntu Linux (14.04, 16.04 and 16.10 IIRC) with binary packages. Then we test compilation from source for debian stretch, centos 6.x,
 fedora 23 and ubuntu 14.04/16.04 (see https://github.com/bazelbuild/bazel/tree/master/src/test/docker) and also for FreeBSD 11 and 12. We also maintain freebsd port of Bazel (@aehlig).